creator-node/scripts/start.sh

```diff
@@ -64,9 +67,18 @@ if [ -z "$dbUrl" ]; then
 fi
 
 if [[ "$devMode" == "true" ]]; then
-    ./node_modules/.bin/nodemon --inspect=0.0.0.0:${debuggerPort} src/index.js | tee >(logger) | ./node_modules/.bin/bunyan
```

Comment:
getting rid of `--inspect`?

Reply:
i'll add this back in

Reply:
this turned out to be much more difficult than expected because apparently nodemon, ts-node and inspect don't play well together. i'll continue to investigate this but don't think this is a blocker for this pr

Reply:
oof gotcha! thanks